test/gui/tst_sharing/test.feature

```diff
@@ -6,7 +6,6 @@ Feature: Sharing
 
     Background:
         Given user "Alice" has been created in the server with default attributes
-        And the setting "shareapi_auto_accept_share" on the server of app "core" has been set to "yes"
```

Comment:
If you are going to remove this one here. then you would need to configure this in oc10 beforehand

Reply:
though auto-accept is removed, the tests are passing but how :thinking:

Reply:
hmm 🤔